tests/upgrades/test_errata.py

```diff
@@ -322,3 +326,270 @@ def test_post_scenario_errata_count_installtion(self):
         )
         for package in FAKE_9_YUM_UPDATED_PACKAGES:
             self._check_package_installed(client_container_id, package)
+
+
+class Scenario_errata_count_with_previous_version_katello_agent(APITestCase):
+    """The test class contains pre and post upgrade scenarios to test erratas count
+    and remotely install using n-1 'katello-agent' on content host.
+
+    Test Steps:
+
+        1. Before Satellite upgrade, Create a content host and register it with
+            Satellite
+        2. Install packages and down-grade them to generate errata.
+        3. Upgrade Satellite
+        4. Check if the Erratas Count in Satellite after the upgrade.
+        5. Install erratas remotely on content host and check the erratas count.
+
+    BZ: 1529682
+    """
+
+    @classmethod
+    def setUpClass(cls):
+        cls.docker_vm = settings.upgrade.docker_vm
+        cls.client_os = DISTRO_RHEL7
+        cls.org = entities.Organization().search(
+            query={'search': 'name="{}"'.format(DEFAULT_ORG)})[0]
+        cls.loc = entities.Location().search(
+            query={'search': 'name="{}"'.format(DEFAULT_LOC)})[0]
+
```

Comment:
any specific reason to add this in default organization and location?

Reply:
@omkarkhatavkar  it used to take the advantage of already synced repos (RHEL 7Server and 6.5 tool repo) as both requires sufficient if create new org/loc ...

Reply:
OK, make sense